Dropdown/DropdownList.js

```diff
@@ -180,142 +180,144 @@ const ReadyState = {
 const DropdownListSpotlightDecorator = hoc((config, Wrapped) => {
 	const WrappedWithRef = WithRef(Wrapped);
 
-	return class extends Component {
-		static displayName = 'DropdownListSpotlightDecorator';
-
-		static propTypes = {
-			/*
-			 * Passed by DropdownBase to resume Spotlight
-			 *
-			 * @type {Function}
-			 */
-			handleSpotlightPause: PropTypes.func,
-
-			/*
-			 * Called when an item receives focus.
-			 *
-			 * @type {Function}
-			 */
-			onFocus: PropTypes.func,
-
-			/*
-			 * Index of the selected item.
-			 *
-			 * @type {Number}
-			 */
-			selected: PropTypes.number
-		};
-
-		constructor (props) {
-			super(props);
-
-			this.clientSiblingRef = createRef(null);
-			this.state = {
-				prevChildren: props.children,
-				prevFocused: null,
-				prevSelected: this.props.selected,
-				prevSelectedKey: getKey(props),
-				ready: isSelectedValid(props) ? ReadyState.INIT : ReadyState.DONE
-			};
-		}
-
-		componentDidMount () {
-			if (this.props.handleSpotlightPause) {
-				this.props.handleSpotlightPause(false);
+	// eslint-disable-next-line no-shadow
+	const DropdownListSpotlightDecorator = (props) => {
```

Comment:
let's rename this component  `DropdownListSpotlight` to avoid eslint rule disabling